src/lib/y2storage/storage_manager.rb

```diff
@@ -366,6 +396,37 @@ def reset_staging_revision
       @staging_revision_after_probing = 0
     end
 
+    # Sanitizes the raw probed devicegraph
+    #
+    # @note The raw probed devicegraph can contain some errors (e.g., incomplete LVM VGs).
```

Comment:
Why is all the description nested into the `@note` tag. I would not use `@note` here, this is just explaining this method.